phoenix-core/src/main/java/org/apache/phoenix/schema/ValueSchema.java

```diff
@@ -261,6 +262,11 @@ public void readFields(DataInput input) throws IOException {
                 typeOrdinal *= -1;
                 this.isNullable = true;
             }
+
+            //if(JDBCType //typeOrdinal )
+
+            PDataType[] typeValues = PDataType.values();
```

Comment:
what's this for?

Reply:
Remnants of when i was trying to push down the handling of the blob type elsewhere rather than my 71 sized array hack.  I'll remove.